sycl/doc/SharedLibraries.md

```diff
@@ -269,6 +269,22 @@ in the `Name` field of the property.
 List of exported symbols is represented in the same way, except the
 corresponding set has the name `SYCL/exported symbols`.
 
+#### Function pointers support
+
+In order to support co-existence of device function pointers feature together
+with dynamic linking, after device code split process `sycl-post-link`
+collects uses of pointers to functions and definitions of functions with
+`indirectly_callable` attribute and attaches info about them to corresponding
+device images via device image properties mechanism.
+Each device image property contains info about signatures of functions that
+were marked with `indirectly_callable` attribute or were used via pointer in
+a form of a string. Example of such string for a function that returns and
+accepts 32bit integer: `i32 (i32)`.
+List of recorded function signatures is represented as a single property set
+with name `SYCL/indirectly called` recorded in the `Name` field of property set.
```

Comment:
Isn't there a need for two lists:
 - the one which says that this image contains kernels which indirectly call functions with these signatures
 - the other one which says that functions of these signatures in this image can be indirectly called?